qp_qiime2/__init__.py

```diff
@@ -70,3 +70,24 @@
     pcoa, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
+
+
+# Define the beta_correlation command
+req_params = {'i-distance-matrix': ('artifact', ['distance_matrix']),
+              'm-metadata-file': ('artifact', ['BIOM'])}
```

Comment:
I'm a bit confused about requesting a BIOM for the metadata-file. I would not request any metadata here, and I will ask the analysis of the artifact for the metadata.